FIRO_TSEnsembles/src/main/java/hec/metrics/MetricCollection.java

```diff
@@ -12,40 +12,37 @@ public class MetricCollection {
     public MetricCollectionTimeSeries parent;
     private Configuration _configuration;
     private float[][] metrics;
-    private Statistics[] metric_statistics;
+    private String metric_statisticsLabel;
 
-    public MetricCollection(Configuration c, Statistics[] statistics, float[][] values)
+    public MetricCollection(Configuration c, String statisticsLabel, float[][] values)
     {
         this._configuration = c;
-        this.metric_statistics = statistics;
+        this.metric_statisticsLabel = statisticsLabel;
         this.metrics = values;
     }
-    public MetricCollection(ZonedDateTime issueDate,ZonedDateTime startDate, Statistics[] statistics, float[][] values)
+    public MetricCollection(ZonedDateTime issueDate,ZonedDateTime startDate, String statisticsLabel, float[][] values)
     {
-        this(new MetricsConfiguration(issueDate,startDate), statistics, values);
+        this(new MetricsConfiguration(issueDate,startDate), statisticsLabel, values);
     }
 
-    public Statistics[] getMetricStatistics() {
-        return metric_statistics;
+    public  String getMetricStatistics() {
+        return metric_statisticsLabel;
     }
 
+    //TODO: fix this to be specific to string label. Contains will catch on first instance every time if multiple.
```

Comment:
I don't see any usages of this method.  Perhaps used by a plugin?

Reply:
I don't really understand it's use case. @HenryGeorgist is there a reason to keep this method around?